src/shared/components/Navigation/Navigation.tsx

```diff
@@ -29,6 +32,62 @@ const Navigation: FunctionComponent<NavigationProps> = ({
 	isOpen,
 	handleMenuClick = () => {},
 }) => {
+	const [areDropdownsOpen, setDropdownsOpen] = useState<{ [label: string]: boolean }>({});
+
+	useEffect(() => {
+		// Init areDropdownsOpen for all navigation items to false
+		// otherwise for some reason the dropdown starts in the open state
+		const openStates = { ...areDropdownsOpen };
+		setDropdownsOpen(openStates);
+		primaryItems.forEach(item => (openStates[item.label] = false));
+		secondaryItems.forEach(item => (openStates[item.label] = false));
+		setDropdownsOpen(openStates);
+	}, []);
+
+	const setDropdownOpen = (label: string, isOpen: boolean): void => {
+		const openStates = { ...areDropdownsOpen };
+		openStates[label] = isOpen;
+		setDropdownsOpen(openStates);
+	};
+
+	const renderNavLinkItem = (item: NavigationItem, className: string, exact: boolean) => {
+		return (
+			<li key={`${item.location}-${item.label}`}>
+				{!!item.location && (
+					<NavLink
+						to={item.location}
+						className={className}
+						activeClassName="c-nav__item--active"
+						exact={exact}
+					>
+						{item.icon && <Icon name={item.icon} />}
+						{item.label}
+					</NavLink>
+				)}
+				{!!item.component && (
+					<Dropdown
+						menuWidth="fit-content"
+						isOpen={areDropdownsOpen[item.label]}
```

Comment:
`isOpen={areDropdownsOpen[item.label] || false}`

Reply:
yep, heb ik reeds gefixed in een toekomstige PR